netci.groovy

```diff
@@ -219,6 +219,8 @@ commitPullList.each { isPr ->
       def myJob = job(jobName) {
         description("Windows ${configuration} tests on ${buildTarget}")
         steps {
+          batchfile('tasklist /FI "IMAGENAME eq devenv.exe"')
+          batchfile('taskkill /FI "IMAGENAME eq devenv.exe"')
```

Comment:
You can do /IM as a shortcut for image name filters. You might also want to do /f to use the force. Otherwise I think it tries to gracefully shutdown and we don't want that.